tests/pipeline_test.py

```diff
@@ -1197,3 +1197,8 @@ def process(
     b[RawData[Sample]] = 7
     assert a.compute(Result) == 29
     assert b.compute(Result) == 53
+
+
+def test_html_repr() -> None:
+    pipeline = sl.Pipeline([make_int], params={float: 5.0})
```

Comment:
Can we add generic providers and a param table for this (or separate) test, just to make sure this doesn't break the repr?